src/ScriptTools.sol

```diff
@@ -40,6 +40,19 @@ library ScriptTools {
         string memory chainInputFolder = string(abi.encodePacked("/script/input/", vm.toString(getRootChainId()), "/"));
         return vm.readFile(string(abi.encodePacked(root, chainInputFolder, input, ".json")));
     }
+    
+    /**
+     * @notice Use standard environment variables to load config.
+     * @dev Will first check FOUNDRY_SCRIPT_CONFIG_TEXT for raw json text.
+     *      Falls back to FOUNDRY_SCRIPT_CONFIG for a standard file definition.
+     *      Finally will fall back to the given string.
+     */
+    function loadConfig(string memory name) internal returns (string memory config) {
+        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", string(""));
+        if (eq(config, "")) {
+            config = readInput(vm.envOr("FOUNDRY_SCRIPT_CONFIG", name));
+        }
```

Comment:
Would the following work?

```suggestion
        config = vm.envOr("FOUNDRY_SCRIPT_CONFIG_TEXT", readInput(vm.envOr("FOUNDRY_SCRIPT_CONFIG", name)));
```

Reply:
Yes this should work. The issue before was that the second part would be evaluated which reverted before it was converted into the optional version. This should work in the one line.